scripts/create-releasenotes.sh

```diff
@@ -2,7 +2,19 @@
 
 # use to assemble the details for our release notes and write them to a file
 #
-# Usage: create-releasenotes.sh pr_num pr_url pr_title commit_id commit_url
+# Usage: create-releasenotes.sh merge_sha
 
-sed "s/PRNUM/$1/;s/PRURL/$2/;s/PRTITLE/$3/;s/COMMITID/$4/;s/COMMITURL/$5/" < gh_release_notes.in > gh_release_notes
+json=$(gh pr list -s merged -S "$1" --json title,number,url)
 
+cp gh_release_notes_top gh_release_notes
```

Comment:
`gh_release_notes_top` now contains the entire message template - I think this needs to be shortened.

Reply:
eh.